packages/docusaurus-init/templates/classic/docs/markdown-features.mdx

```diff
@@ -0,0 +1,250 @@
+</Highlight> are my favorite colors.
+
+## Tabs
+
+Docusaurus provides `<Tabs>` component that you can use to show tabbed contents.
+
+```jsx
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="react"
+  values={[
+    {label: 'React', value: 'react'},
+    {label: 'Vue', value: 'vue'},
+    {label: 'Angular', value: 'angular'},
+  ]}>
+  <TabItem value="react">
+    This is <em>React</em>
+  </TabItem>
+  <TabItem value="vue">
+    This is <em>Vue</em>
+  </TabItem>
+  <TabItem value="angular">
+    This is <em>Angular</em>
+  </TabItem>
+</Tabs>;
+```
+
+The component above will render the following:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="react"
+  values={[
+    {label: 'React', value: 'react'},
+    {label: 'Vue', value: 'vue'},
+    {label: 'Angular', value: 'angular'},
+  ]}>
+  <TabItem value="react">
+    This is <em>React</em>
+  </TabItem>
+  <TabItem value="vue">
+    This is <em>Vue</em>
+  </TabItem>
+  <TabItem value="angular">
+    This is <em>Angular</em>
+  </TabItem>
+</Tabs>
+
+:::info
+
+You can load each tab lazily by passing the `lazy` prop to the `Tabs` component.
+
+:::
+
+## Inline Table of Contents
+
+If you didn't notice already, each markdown document displays a tab of content on the top-right corner.
+
+You can display those table of contents inside a markdown document, inline by using the `<TOCInline>` component.
+
+```jsx
+import TOCInline from '@theme/TOCInline';
+
+<TOCInline toc={toc} />;
+```
+
+will render the following:
+
+import TOCInline from '@theme/TOCInline';
+
+<TOCInline toc={toc} />
+
+## Callouts/Admonitions
+
+Docusaurus supports adminitions that you can utilize below:
+
+    :::note
+
+    The content and title *can* include markdown.
+
+    :::
+
+    :::tip You can specify an optional title
+
+    Heads up! Here's a pro-tip.
+
+    :::
+
+    :::info
+
+    Useful information.
+
+    :::
+
+    :::caution
+
+    Warning! You better pay attention!
+
+    :::
+
+    :::danger
+
+    Danger danger, mayday!
+
+    :::
+
+:::note
+
+The content and title _can_ include markdown.
+
+:::
+
+:::tip You can specify an optional title
+
+Heads up! Here's a pro-tip.
+
+:::
+
+:::info
+
+Useful information.
+
+:::
+
+:::caution
+
+Warning! You better pay attention!
+
+:::
+
+:::danger
+
+Danger danger, mayday!
+
+:::
+
+## Code Blocks
+
+You can showcase your code better when you are using code block as it will format and provide syntax highlighting for your code blocks.
+
+    ```jsx title="src/components/HelloDocusaurus.js"
+    function HelloDocusaurus() {
+        return (
+            <h1>Hello, Docusaurus!</h1>
+        )
+    }
+    ```
+
+The markdown above will render the code block with `jsx` syntax highlighting and show the provided title:
+
+```jsx title="src/components/HelloDocusaurus.js"
+function HelloDocusaurus() {
+  return <h1>Hello, Docusaurus!</h1>;
+}
+```
+
+## Assets
```

Comment:
can be removed, niche MD feature

however we should display simpler usage of md images